scripts/pcp/get_transcripts.pl

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env perl
+
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+=head1 NAME
+  get_transcripts.pl
+=head1 DESCRIPTION
+Retrieves the DNA for all transcripts from a given database and dumps out to fasta format file.
+=head1 OPTIONS
+=head1
+=head2 Database connection options
+  -dbhost   Host name for database.
+  -dbport   What port to connect to.
+  -dbname   What name to connect to.
+  -dbuser   What username to connect as.
+  -dbpass   What password to use.
+
+  -dna_dbhost   Host name for database.
+  -dna_dbport   What port to connect to.
+  -dna_dbname   What name to connect to.
+  -dna_dbuser   What username to connect as.
+  -dna_dbpass   What password to use.
+
+=head2 Other options
+  --coords coordinate system name to use, default toplevel
+=head1 EXAMPLES
+  ./get_transcripts.pl --dbhost genebuild5 \
+    --dbuser ensro \
+    --dbname <database_name> \
+    --dna_dbhost genebuild6 \
+    --dna_dbname <dna_database_name> \
+    > output.fasta
+=cut
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+
+use Bio::EnsEMBL::DBSQL::DBAdaptor;
+
+## Some defaults
+
+my $coord_system = 'toplevel';
+
+my $dbname = '';
+my $user   = 'ensro';
+my $host = $ENV{GBS5};
+my $port = $ENV{GBP5};
```

Comment:
Can this be set to '' instead of fetching the value from an ENV? People might think they need these ENVs

Reply:
Done.